apps/chatbots/urls.py

```diff
@@ -65,4 +65,19 @@
         views.chatbot_chat_embed,
         name="chatbot_chat_embed",
     ),
+    path(
+        "<int:experiment_id>/settings/edit-mode/",
+        views.settings_edit_mode,
+        name="chatbots_settings_edit_mode",
```

Comment:
nit: drop the `chatbots_` prefix from urls since they are already namespaced: `chatbots:chatbots_settings_edit_mode`
```suggestion
        name="settings_edit_mode",
```

Same below.